### PR TITLE
Add more instructions for Spring Boot Hello World.

### PR DESCRIPTION
**Pull request by tswast**
This sample is linked to from https://cloud.google.com/java/

### CODE REVIEW

**Comment by lesv** [comment]
Tim - I'm going to review this and the rest when I no longer have any meds in my system.  I think I'd like us to find a standard format for README's.

**Comment by tswast** [comment]
Agreed. A standard would be good.

File: helloworld-springboot/README.md [hunk 1]
```diff
@@ -1,17 +1,83 @@
...
+This sample shows how to run a [Spring Boot][spring-boot] application on [Google
+Cloud Platform][cloud-java]. It uses the [Google App Engine flexible
+environment][ae-docs].
 
-## Run the Application locally
-1. Set the correct Cloud SDK project via `gcloud config set project YOUR_PROJECT`
-id of your application.
-2. Run `mvn spring-boot:run`
-4. Visit http://localhost:8080
+[ae-docs]: https://cloud.google.com/appengine/docs/flexible/
```
Comment (lesv) [inline helloworld-springboot/README.md]: `[App Engine-Flexible]`

File: helloworld-springboot/README.md [hunk 2]
```diff
@@ -1,17 +1,83 @@
...
 
-5. `mvn gcloud:deploy`
-6. Visit `http://YOUR_PROJECT.appspot.com`.
+## Before you begin
+
+This sample assumes you have [Java 8][java8] installed.
+
+[java8]: http://www.oracle.com/technetwork/java/javase/downloads/
+
+### Download Maven
```
Comment (lesv) [inline helloworld-springboot/README.md]: I really like this!

File: helloworld-springboot/README.md [hunk 3]
```diff
@@ -1,17 +1,83 @@
...
+
+[enable-billing]: https://console.cloud.google.com/project/_/settings
+
+### Install the Google Cloud SDK.
+
+If you haven't already installed the Google Cloud SDK, [install and initialize
+the Google Cloud SDK][cloud-sdk] now. The SDK contains tools and libraries that
+enable you to create and manage resources on Google Cloud Platform.
+
+[cloud-sdk]: https://cloud.google.com/sdk/
```
Comment (lesv) [inline helloworld-springboot/README.md]: `gcloud components update app-engine-java`
`gcloud components update`

File: helloworld-springboot/README.md [hunk 4]
```diff
@@ -1,17 +1,83 @@
...
+
+### Install the Google App Engine SDK for Java
+
+Run
```
Comment (lesv) [inline helloworld-springboot/README.md]: Not sure Run is required here.

**Comment by lesv** [comment]
PTAL

File: helloworld-springboot/README.md (hunk 1 above)
Comment (tswast) [inline helloworld-springboot/README.md]: > `[App Engine-Flexible]`

Done.

File: helloworld-springboot/README.md (hunk 3 above)
Comment (tswast) [inline helloworld-springboot/README.md]: I added `gcloud components update`.

File: helloworld-springboot/README.md (hunk 4 above)
Comment (tswast) [inline helloworld-springboot/README.md]: Removed "Run"

**Comment by tswast** [comment]
Updated. @lesv PTAL.

**Comment by lesv** [comment]
LGTM

**Pull request merged**